dwm1000sensor:
  num_drones: 6